account_invoice_mgmt_dms/models/account_move.py

```diff
@@ -142,3 +146,23 @@ def action_invoice_register_payment(self):
                     _("Must be approved before making payment")
                 )
         return super(AccountMove, self).action_invoice_register_payment()
+
+    def action_approve_complete_processing(self):
+        invoice_ids = self.env["account.move"].browse(self._context.get("active_ids", []))
+        for record in invoice_ids.filtered(lambda x: x.complete_proceesing_id and x.complete_proceesing_id.state_account_move == 'pending'):
```

Comment:
Is `approve_account_move()` a `multi` method? If it is, we can call it together for every filtered record

Reply:
Its not multi